tests/schemas/api_v3/event.schema.js

```diff
@@ -29,12 +29,12 @@ const block = {
 
 const eventSchema = {
 	id: Joi.string().required(),
-	moduleID: Joi.string().required(),
-	moduleName: Joi.string().required(),
-	typeID: Joi.string().required(),
-	data: Joi.string().required(),
-	topics: Joi.array().required(),
-	block: Joi.object(block).optional(),
+	module: Joi.string().pattern(regex.MODULE).required(),
+	name: Joi.string().required(),
```

Comment:
Can we please add the pattern check for `name` as well?

Reply:
applied.